src/providers/RelayProvider.ts

```diff
@@ -30,6 +30,15 @@ export class RelayProvider implements TLProvider {
     return utils.fetchUrl<T>(`${this.relayApiUrl}/${trimmedEndpoint}`, options)
   }
 
+  public async fetchPostEndpoint<T>(endpoint: string, data: any): Promise<T> {
```

Comment:
I would just call it `postEndpoint` or `postToEnpoint`. I think the fetch is redundant.

Reply:
Sounds good